assets/javascripts/test_result.js

```diff
@@ -96,6 +96,27 @@ function checkPreviewVisible(stepPreviewContainer, preview) {
   }
 }
 
+function XYRes() {
+  // the test VM's resolution
+  let x_res;
+  let y_res;
+  try {
+    var xhr = new XMLHttpRequest();
+    var test_file =  "/api/v1/jobs/"+testStatus.jobid;
+    xhr.open("GET", test_file, false);
+    xhr.send(null);
+    let json = JSON.parse(xhr.responseText);
+    x_res = json.job.settings.XRES;
+    y_res = json.job.settings.YRES;
+  }
+  catch(error) {
+    // if {X,Y}RES not defined, will be using the defaults
+    x_res = 1024;
```

Comment:
Ignoring errors silently is not very nice.

Reply:
Seconded. Only ignore very specific errors.

Reply:
Hi @Martchus, thanks a lot for all this feedback. This is my first PR on openQA I wouldn't be surprised if my solution isn't an optimal one. RE the original API requests: could you point me to a concrete example?

I also agree that this error handling isn't ideal. Maybe I could catch specific "{X,Y}_RES doesn't exist" errors with this arm, and report other errors as "real" errors?

Reply:
It's customary to handle the errors you're interested in and just rethrow what remains.